src/adapter/variableStore.ts

```diff
@@ -1027,14 +993,28 @@ class VariablesMap {
 @injectable()
 export class VariableStore {
   private vars = new VariablesMap();
+  private readonly contextSettings: IContextSettings;
 
   constructor(
     @inject(IRenameProvider) private readonly renameProvider: IRenameProvider,
     @inject(ICdpApi) private readonly cdp: Cdp.Api,
     @inject(IDapApi) private readonly dap: Dap.Api,
     @inject(AnyLaunchConfiguration) private readonly launchConfig: AnyLaunchConfiguration,
     private readonly locationProvider: IVariableStoreLocationProvider,
-  ) {}
+  ) {
+    this.contextSettings = {
```

Comment:
Previously we were code-genning every time we made a preview for a variable. Fortunately Acorn is pretty speedy, but we can precompute that instead easily :)